feeder/src/main/java/org/n52/sos/importer/feeder/Feeder.java

```diff
@@ -342,6 +349,11 @@ public void importData(final DataFile dataFile)
             LOG.error("No measured value columns found in configuration");
             return;
         }
+        // Get stored lastUsedTimestamp, if it is set:
```

Comment:
@MojioMS This comments seems misleading as the next line is setting the lastLine value and not lastUsedTimestamp! Please update the comment or fix the bug.